packages/atomic/src/components/facets/atomic-category-facet/atomic-category-facet.pcss

```diff
@@ -2,17 +2,12 @@
 @import '../facet-common.pcss';
 @import '../facet-search/facet-search.pcss';
 
-.parent-button,
 .parent-active {
-  @apply w-full py-2.5 pr-2 pl-7 text-left;
+  @apply pl-7;
 }
```

Comment:
`.parent-active` is now a facet value just like the other ones, inheriting all their styling.